components/footer.tsx

```
@@ -21,27 +39,107 @@ export const Footer = () => {
-          <a href="">Guestbook</a>
-          <a href="">Sitemap</a>
-          <a href="">Github</a>
+        <div className="text-sm md:text-base text-muted-foreground mt-10 md:mt-20 space-x-5">
+          <button
+            className="cursor-pointer hover:text-foreground"
+            onClick={() => setOpen(true)}
+          >
+            Quick Links
+          </button>
+          <a
+            target="_blank"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)